bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/base/ZeroMagnitudeStar.java

```diff
@@ -8,38 +8,38 @@
  */
 public final class ZeroMagnitudeStar {
     // Keep anyone from instantiating this object.
-    private ZeroMagnitudeStar() {
-    }
+    private ZeroMagnitudeStar() {}
 
     /**
      * Get average flux in photons/s/m^2/nm in specified waveband.
      * Overrides method in base class AverageFlux.
      */
-    public static int getAverageFlux(WavebandDefinition waveband) {
+    public static int getAverageFlux(final MagnitudeBand band) {
 
         // flux densities for each waveband for zero-magnitude star (of some type)
         // units are photons/s/m^2/nm
-        switch (waveband) {
-            case U:     return 75900000;
-            case B:     return 146100000;
-            case V:     return 97100000;
-            case R:     return 64600000;
-            case I:     return 39000000;
-            case J:     return 19700000;
-            case H:     return 9600000;
-            case K:     return 4500000;
-            case L:     return 990000;
-            case M:     return 510000;
-            case N:     return 51000;
-            case Q:     return 7700;
+        // TODO: Missing support for Y band
+        if      (band.equals(MagnitudeBand.U$.MODULE$))  return 75900000;
```

Comment:
Do you think this could go into a scala wrapper, it should be nicer in a `match` format